### DISCUSSION TITLE
What is a Cabal?

### DISCUSSION THREAD

**Discussion by Psypher9** [rezalas/riftshadow#195] [Q&A] [answered]
I've been looking around at the code and bit and it seems like a Cabal is like a guild.

Is that true?

Are Cabals player-generated or set by the gameworld?

What can you do with a Cabal and how do they work?

**Comment by rezalas** [comment, marked as answer]
Yep cabals are guilds created in the game. They're pre-generated and have special quests, powers, and ranks within each cabal. They cannot be player made at this time, though that's an interesting concept I haven't seen in other MUDs.
